src/Components/Notifications/NotificationsEmptyStateByType.tsx

```diff
@@ -22,6 +22,10 @@ const emptyStateByType: Record<
     message:
       "Create alerts on an artist or artwork page and get notifications here when there’s a match.",
   },
+  following: {
+    title: "",
+    message: "",
```

Comment:
Will replace with real copy when we clarify it.

Reply:
Updated copy for "Following", will update "All" copy async once there is a decision.